packages/api/src/util/sort-object.ts

```diff
@@ -0,0 +1,13 @@
+const sortObject = (obj: any): any => {
```

Comment:
did you write this file or copy from somewhere? if copied you should put a link to the source

Reply:
custom code inspired by https://github.com/luniehq/cosmos-api/blob/c4e7a5be4c2ed2e2069b2556fcda69285b6bb52b/src/signature.js#L52 but changed a lot